utilities/random_number_generation/primes.py

```diff
@@ -0,0 +1,44 @@
+from Crypto.Util import number
+
+"""
```

Comment:
Thinking whether or not it would be better to wrap all these methods in a class named `CustomPrime` or something similar and call them at the class level in other files so that its more obvious where they came from. So for example in `bbs.py` you would do `from primes import CustomPrime` and then in `getModulus` have `CustomPrime.getStrongBlumSize(psize)`

Reply:
Methods `isStrongPrime`, `isBlumPrime`, `isStrongBlumPrime`, `getPrime` seem like good private methods while the rest seem like good public ones.

Reply:
I like this idea a lot; definitely going to do this.

Reply:
I ended up splitting the isPrime functions and the getPrime functions into separate classes because the isPrime functions might be useful for stuff in the future

Reply:
Looks really good bro lets go